gui/src/app/pages/HomePage/Sidebar.tsx

```diff
@@ -62,7 +60,8 @@ const Sidebar: FunctionComponent<Sidebar> = ({
       {/* For spacing purposes */}
       <Toolbar />
 
-      <h3>Examples</h3>
+
+      <div className="SidebarContentWrapper SidebarHeading">Examples</div>
```

Comment:
Is there a semantic reason to prefer (or not prefer) a styled div vs h3?

Reply:
Technically there are accessibility reasons to prefer `H` tags where appropriate, but I didn't bring it up because a) we haven't been paying a lot of attention to that overall and b) it's not clear to me that it would actually be appropriate here anyway, as that sort of advice is really for a different sort of page/site...